content/en/blog/_posts/2019-06-13-volume-cloning-alpha.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: blog
+title: 'Introducing Volume Cloning Alpha for Kubernetes'
+date: 2019-06-13
+---
+
+**Author**: John Griffith (Red Hat)
+
+Kubernetes v1.15 introduces alpha support for volume cloning. This feature allows you to create new volumes using the contents of existing volumes in the users namespace using the Kubernetes API.
+
+## What is a Clone?
+
+Many storage systems provide the ability to to create a "clone" of a volume.  A clone is a duplicate of an existing volume that is its own unique volume on the system, but is an exact duplicate of in terms of its contents.  A clone is similar to a snapshot in that it's a point in time copy of a volume, however rather than creating a new snapshot object from a volume, we're instead creating a new independent volume, sometimes thought of as pre-populating the newly created volume.
```

Comment:
s/is an exact duplicate of in terms of its contents/has duplicate contents of the source volume/